### PR TITLE
feature: the tools `Util / reindex` trimmed the extra blank line at t…

### PR DESCRIPTION
**Pull request by membphis**
feature: the tools `Util / reindex` trimmed the extra blank line at the end.

I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

File: util/reindex [hunk 1]
```diff
@@ -44,7 +44,9 @@ sub reindex {
         push @lines, $_;
     }
     close $in;
+
     my $text = join '', @lines;
+    $text =~ s/\n+$/\n/;
```
Comment (agentzh) [inline util/reindex]: I think `$text =~ s/\n+\z/\n/sm;` is better.

Comment (membphis) [inline util/reindex]: Your version is much better, changed.

**Comment by membphis** [comment]
I run the new version of `reindex`, it helped me fix old test cases.

**Comment by agentzh** [comment]
Merged. Thanks.

**Comment by agentzh** [comment]
@membphis Okay, `s/\n\n+\z/\n/sm` is actually even better :)

**Pull request closed without merging**